types/TestResult.js

```diff
@@ -100,20 +100,20 @@ export type FormattedAssertionResult = {
 
 export type AggregatedResult = {
   coverageMap?: ?CoverageMap,
-  numFailedTests: number,
-  numFailedTestSuites: number,
-  numPassedTests: number,
-  numPassedTestSuites: number,
-  numPendingTests: number,
-  numPendingTestSuites: number,
-  numRuntimeErrorTestSuites: number,
-  numTotalTests: number,
-  numTotalTestSuites: number,
-  snapshot: SnapshotSummary,
-  startTime: number,
+  numFailedTests?: number,
+  numFailedTestSuites?: number,
+  numPassedTests?: number,
+  numPassedTestSuites?: number,
+  numPendingTests?: number,
+  numPendingTestSuites?: number,
+  numRuntimeErrorTestSuites?: number,
+  numTotalTests?: number,
+  numTotalTestSuites?: number,
+  snapshot?: SnapshotSummary,
+  startTime?: number,
   success: boolean,
-  testResults: Array<TestResult>,
-  wasInterrupted: boolean,
+  testResults?: Array<TestResult>,
+  wasInterrupted?: boolean,
```

Comment:
I'm trying to update [this callback](https://github.com/sandersky/jest/blob/eaa01e939d2865037d40ad6374691c2da0664ef3/packages/jest-cli/src/runJest.js#L155) to get an `AggregatedResult` as it's argument but under [this scenario](https://github.com/sandersky/jest/blob/eaa01e939d2865037d40ad6374691c2da0664ef3/packages/jest-cli/src/runJest.js#L179) the object only contains the `success` key. I guess another alternative may be to have that scenario return a filled out object with the other required properties. Thoughts?